pandas/core/ops/__init__.py

```diff
@@ -59,6 +59,37 @@
     rxor,
 )
 
+# -----------------------------------------------------------------------------
+# constants
+ARITHMETIC_BINOPS: Set[str] = {
+    "add",
+    "sub",
+    "mul",
+    "pow",
+    "mod",
+    "floordiv",
+    "truediv",
+    "divmod",
+    "radd",
+    "rsub",
+    "rmul",
+    "rpow",
+    "rmod",
+    "rfloordiv",
```

Comment:
is there a reason these are not fixtures?

Reply:
> is there a reason these are not fixtures?

This is used in *code*, not in tests